### PR TITLE
Quick Look preview visual updates for small viewports

### PR DESCRIPTION
**Pull request by jtbandes**
**User-Facing Changes**
Improved macOS Quick Look preview when displayed at small sizes (such as in the Get Info window in Finder).

**Description**

- Added `yarn serve:quicklook` for faster development
- Added styled-components transformer (which we already using in our other webpack builds) for better CSS class names
- Added breakpoint for small sizes
- Remove extra table padding at small sizes
- Use some non-breaking spaces to avoid breaking in weird places
- Fixed filtering of `compression` values so that `""` displays as `Compression: none`

Before:

image

After:

Screen Shot 2022-03-01 at 4 59 03 PM Screen Shot 2022-03-01 at 4 59 13 PM

### CODE REVIEW

File: desktop/quicklook/FileInfoDisplay.tsx [hunk 1]
```diff
@@ -50,6 +51,10 @@ const IconContainer = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
+
+  @media (max-width: ${styleConstants.breakpoints.narrowMaxWidth}) {
```
Comment (defunctzombie) [inline desktop/quicklook/FileInfoDisplay.tsx:55]: Quicklook should probably use its own set of breakpoints specific to its needs? Or a semantic name specific to quicklook?

**Review by defunctzombie — APPROVED** [review]

File: desktop/quicklook/FileInfoDisplay.tsx (hunk 1 above)
Comment (jtbandes) [reply desktop/quicklook/FileInfoDisplay.tsx:55]: Not sure I understand the suggestion; this styleConstants does live inside the quicklook folder.

Comment (defunctzombie) [reply desktop/quicklook/FileInfoDisplay.tsx:55]: I missed that - somehow thought it was coming from a style constants in studio-base. I now see I confused that with `sharedStyleConstants.ts` 🤦

Comment (jtbandes) [reply desktop/quicklook/FileInfoDisplay.tsx:55]: shared, but not _too_ shared

**Pull request merged**